assemblies/pentaho-war/pom.xml

```diff
@@ -840,8 +840,9 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>org.hitachivantara</groupId>
```

Comment:
The library snakeyaml-2.2.jar ( "org.yaml:snakeyaml:2.2") replaces the previous interations of safeyaml-1.34.1.jar ( "org.hitachivantara:safeyaml" and "com.konloch:safeyaml") and snakeyaml-1.33.jar (""org.yaml:snakeyaml:1.33")

The artifact `snakeyaml.jar` is a runtime dependency on `pentaho-platform-exentions.jar` and `pentaho-platform-repository.jar`

For pentaho reporting the "library" folders:
- PS-CE (before)
  - server/pentaho-server/tomcat/webapps/pentaho/WEB-INF/lib/pentaho-platform-extensions.jar
  - **server/pentaho-server/tomcat/webapps/pentaho/WEB-INF/lib/safeyaml.jar**
- PS-CE (after)
  - server/pentaho-server/tomcat/webapps/pentaho/WEB-INF/lib/pentaho-platform-extensions.jar
  - **server/pentaho-server/tomcat/webapps/pentaho/WEB-INF/lib/snakeyaml.jar**

PS-CE feeds into PS-EE so  no need to duplicate this configuration in PS-EE pom.xmls